pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -115,16 +115,12 @@ func fixUserPartOfHyperVAdmins() (bool, error) {
 	}
 	groupName := strings.TrimSpace(strings.Replace(strings.TrimSpace(outGroupName), "BUILTIN\\", "", -1))
 
-	outUsername, _, err := powershell.Execute(`Write-Host $env:USERNAME`)
-	if err != nil {
-		return false, errors.New("Unable to get user name")
-	}
-	username := strings.TrimSpace(outUsername)
+	username := os.Getenv("USERNAME")
 
 	netCmdArgs := fmt.Sprintf(`([adsi]"WinNT://./%s,group").Add("WinNT://%s,user")`, groupName, username)
 	_, _, err = powershell.ExecuteAsAdmin(netCmdArgs)
 	if err != nil {
-		return false, errors.New("Error adding user to group")
+		return false, errors.New("Unable to get user name")
```

Comment:
@teuf, why you mention the username here?

Reply:
I guess this is in case `os.Getenv` returned an empty string, this would fail..

Reply:
It's just that I forgot to change the error message, this block was preexisting before the commit.